mini_api.py

```diff
@@ -12,59 +21,49 @@
     "django.contrib.auth",
     "rest_framework",
 )
-# helper function to locate this dir
-def here(x):
-    return os.path.join(os.path.abspath(os.path.dirname(__file__)), x)
-
-
-TEMPLATES = [
-    {
-        "BACKEND": "django.template.backends.django.DjangoTemplates",
-        "DIRS": here("."),
-        "APP_DIRS": True,
-    }
-]
-
+TEMPLATES = [{"BACKEND": "django.template.backends.django.DjangoTemplates",
+              "DIRS": [BASE_DIR,], "APP_DIRS": True}]
 STATIC_URL = '/static/'
-STATICFILES_DIRS = (
-    here('static'),
-)
-
-
+STATICFILES_DIRS = (join(BASE_DIR, 'static'),)
 
+SETTINGS = dict((key,val) for key, val in locals().items() if key.isupper())
```

Comment:
Yes, checking for is `key.isupper()` is a bit better than repeating another `SETTINGS` dict

Reply:
I looked at the Django code and that's the check they do to raise the error so I was like... I can make that check pass ;)